docs/vector-db-schema-design.md

```diff
@@ -0,0 +1,136 @@
+# Weaviate  Database Schema Documentation
+
+## Overview
+
+This document describes the vector database schema, indexing strategies, and design decisions for a tourism attraction management system built on Weaviate. 
+
+## Schema Design
+
+### Primary Class: `Attraction`
+
+The main class that stores all place-related data with unified schema for attractions, restaurants, cafes, etc.
+
+#### Properties
+
+| Property                         |           Type | Storage / Example (Weaviate-style)                                | Tokenization / Notes                            |
+| -------------------------------- | -------------: | ----------------------------------------------------------------- | ----------------------------------------------- |
+| `name`                           |           text | `"Café Example"`                                                  | `word` — searchable place name                  |
+| `description`                    |           text | long text                                                         | `word` — full-text description                  |
+| `chunk_id`                       |           text | `"page_001_chunk_03"`                                             | -                                               |
+| `address`                        |           text | `"123 Main St, Kyiv, Ukraine"`                                    | - (human-readable formatted\_address)           |
+| `postal_code`                    |           text | `"01001"`                                                         | - (from address\_components)                    |
+| `administrative_area_level_1`    |           text | `"Kyiv"`                                                          | -                                               |
+| `administrative_area_level_2`    |           text | `"Kyiv City"`                                                     | -                                               |
+| `sublocality_level_1`            |           text | `"Podil"`                                                         | -                                               |
+| `coordinates`                    | geoCoordinates | `{ "latitude": 50.4501, "longitude": 30.5234 }`                   | geo — for geo queries (in future)                           |
```

Comment:
The document mentions future geo queries. If geo-search is a priority, consider adding sample queries and index configuration for geospatial search.